app/main.py

```diff
@@ -1,3 +1,76 @@
+from __future__ import annotations
+
+from typing import Union
+
+
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: int or float) -> None:
```

Comment:
The type hint `int or float` should be replaced with `Union[int, float]` for consistency with the rest of the code and to follow standard type hinting practices.